app/src/main/kotlin/com/wire/android/ui/common/DeletedLabel.kt

```diff
@@ -1,43 +1,64 @@
 package com.wire.android.ui.common
 
+import androidx.compose.foundation.BorderStroke
 import androidx.compose.foundation.border
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.layout.wrapContentSize
 import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.LocalTextStyle
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.style.TextAlign
-import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
 import com.wire.android.R
-import com.wire.android.ui.theme.wireTypography
+import com.wire.android.ui.theme.wireColorScheme
 
+/**
+ * Outlined box with a text inside.
+ * Used for things like "Deleted" users,
+ * and "Deleted message" or "Edited message"
+ */
 @Composable
-fun DeletedLabel(modifier: Modifier = Modifier) {
-    Box(
-        modifier = modifier.border(
-            width = dimensions().spacing1x,
-            shape = RoundedCornerShape(dimensions().spacing4x),
-            color = colorsScheme().secondaryButtonDisabledOutline
-        )
+fun StatusBox(
+    statusText: String,
+    modifier: Modifier = Modifier
+) {
+    CompositionLocalProvider(
+        LocalTextStyle provides MaterialTheme.typography.labelSmall
```

Comment:
not sure about the benefit of LocalTextStyle, are we sure we want all the Text() composable that could be used here to apply the same style ? If not this will add more overhead I think